src/abstractdataframe/abstractdataframe.jl

```diff
@@ -499,7 +502,7 @@ function get_stats(col::AbstractArray{>:Missing})
     )    
 end 
 
-function get_stats(col)
+function get_stats(col::AbstractArray{String})
```

Comment:
Do we really need these special `String` methods? AFAICT they give the same result as the fallback except for minimum and maximum, right? I'd rather not add a special method just for that, and printing "minimum" and "maximum" strings can make sense (e.g. for ordered categories). At the very least it doesn't really hurt and it's not completely unintuitive for users to use lexicographic ordering.